### ISSUE TITLE
xml path file read as xml content

### ISSUE THREAD

**Issue by sg29info** [akretion/factur-x#26]
Hi,

I'm running into a bug when passing xml path file as an argument like :
`> python3.9 facturxmerge.py ./pdfA_invoice.pdf ./xml.xml`

from my test file :
```
from datetime import datetime
import sys
import os
from facturx import generate_from_file

if __name__ == "__main__":
    pdfFileName = os.getcwd()+"/"+sys.argv[1]
    xmlFileName = os.getcwd()+"/"+sys.argv[2]

    now = datetime.now()
    outputFile = now.strftime("%Y_%m_%d_%H_%m_%S")+"_factur-x.pdf"
    outputFile = os.getcwd()+"/"+outputFile

    generate_from_file(pdfFileName, xmlFileName, output_pdf_file=outputFile)
```

Bug is :
`lxml.etree.XMLSyntaxError: Start tag expected, '<' not found`

In the facturx.py file at row 1106 the code test if the xml arg is a string : if isinstance(xml, (str, bytes)):
and yes, /path/to/my/xml.xml is a string... but not xml, then any further process fail.
Is that intended ?

**Comment by alexis-via** [comment]
Yes, it is intended ; if you give the XML arg as string, it expects to have the content of the XML file as string, not the path to the xml file.
But I understand you point: it is not consistent with the first argument, where you can give the pdf file path as string.
We could try to detect is the string is an XML or a file path and accept both...

**Comment by alexis-via** [comment]
I implemented it in release 3.12.
